engine/src/main/java/org/terasology/engine/modes/GameState.java

```diff
@@ -33,6 +33,8 @@
 
     void dispose();
 
+    void dispose(boolean saveRequested);
```

Comment:
SAme as with changeState: Something like a save system should not have a impact on those generic interfaces.